blockchain/chain_sync/src/network_context.rs

```diff
@@ -93,6 +114,38 @@ impl SyncNetworkContext {
         Ok(fts.remove(0))
     }
 
+    /// Requests that some content with a particular Cid get fetched over Bitswap if it doesn't
+    /// exist in the BlockStore.
+    pub async fn bitswap_get<TMessage: DeserializeOwned>(
+        &self,
+        content: Cid,
+    ) -> Result<TMessage, String> {
+        // TODO: check to see if we have it in our database
```

Comment:
why not just do it in this PR? It's extremely low cost since self has a blockstore